azure_jumpstart_arcbox/ARM/key-vault/key-vault.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Change filename and path from key-vault to keyVault

Reply:
done